### PR TITLE
Fix calling set_measure on a node that didn't previously have a measure func

### LINKED ISSUE #250: Tentative migration to taffy 0.2 failed
## `taffy` version

Targetting ba4cd3c3b1b36f72434469312416814f82bea5ff

## What you did

Attempted to migrate Bevy to Taffy 0.2, via this [commit](https://github.com/bevyengine/bevy/commit/86b592943aff8fd0779c6fece729511bea1a30b5) in this [branch](https://github.com/rparrett/bevy/tree/taffy-2).

Then, run any UI example in Bevy.

## What went wrong

thread 'main' panicked at 'invalid SparseSecondaryMap key used', /Users/me/.cargo/git/checkouts/taffy-3438f0b42e7e0e0f/ba4cd3c/src/node.rs:207:13
   0: std::panicking::begin_panic
             at /rustc/897e37553bba8b42751c67658967889d11ecd120/library/std/src/panicking.rs:616:12
   1: <slotmap::sparse_secondary::SparseSecondaryMap<K,V,S> as core::ops::index::IndexMut<K>>::index_mut
             at /Users/me/.cargo/registry/src/github.com-1ecc6299db9ec823/slotmap-1.0.6/src/sparse_secondary.rs:852:21
   2: taffy::node::Taffy::set_measure
             at /Users/me/.cargo/git/checkouts/taffy-3438f0b42e7e0e0f/ba4cd3c/src/node.rs:207:13
   3: bevy_ui::flex::FlexSurface::upsert_leaf
             at ./crates/bevy_ui/src/flex/mod.rs:110:13

## Additional information

Identified by @rparrett. 

>  it seems like set_measure is just not accounting for the case where there was no previous measure function?

### PR DESCRIPTION
**Pull request by nicoburns**
# Objective

Fixes #250. Also added a unit test.

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]
Nice!

**Review by Weibye — APPROVED** [review]
🚀

**Comment by rparrett** [comment]
Works for me with Bevy

File: src/node.rs [hunk 1]
```diff
@@ -482,6 +482,18 @@ mod tests {
         assert_eq!(taffy.layout(node).unwrap().size.width, 100.0);
     }
 
+    #[test]
+    fn set_measure_of_previously_unmeasured_node() {
+        let mut taffy = Taffy::new();
+        let node = taffy.new_leaf(FlexboxLayout::default()).unwrap();
```
Comment (alice-i-cecile) [inline src/node.rs:488]:
```suggestion
        let node = taffy.new_leaf(Style::default()).unwrap();
```

**Pull request merged**